<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardDlg</class>
 <widget class="QWidget" name="KeyboardDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="KLineEdit" name="leNumber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="5">
    <widget class="KLineEdit" name="leText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="KPushButton" name="pbAdd">
     <property name="text">
      <string>=====&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="KPushButton" name="pbDeleteRight">
     <property name="text">
      <string>LÃ¶schenR</string>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="KPushButton" name="pbOk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="KPushButton" name="pbShift">
     <property name="text">
      <string>Umschalt</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KPushButton" name="pbCapslock">
     <property name="text">
      <string>Feststell</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="KPushButton" name="pbNextPage">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="lbPage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Seite 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="KPushButton" name="pbPreviousPage">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="9">
    <layout class="QGridLayout" name="glButtonForm"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
